dagql/idtui/frontend.go

```diff
@@ -24,6 +25,13 @@ import (
 	sdklog "go.opentelemetry.io/otel/sdk/log"
 )
 
+// having a bit of fun with these. cc @vito @jedevc
+var skipLoggedOutTraceMsgEnvs = []string{"NOTHANKS", "SHUTUP", "GOAWAY", "STOPIT"}
+
+//nolint:gosec
+var loggedOutTraceMsg = fmt.Sprintf("Setup full trace at %%s.\nTo turn this off, export %s=1",
```

Comment:
```suggestion
var loggedOutTraceMsg = fmt.Sprintf("Setup full trace at %%s. To hide this: export %s=1",
```

Shorten + keep on one line

Reply:
Yup, yup. We should definitely keep this to one line :tada:

However, even with this suggestion, it's *still* 87 characters (with the longest `NOTHANKS`), so can overflow a single standard terminal line.

We could trim it further, and remove ` this` (gets us to 82), and potentially change `Setup` to `See` (which brings us down to 80 perfectly).

```go
var loggedOutTraceMsg = fmt.Sprintf("See full trace at %%s. To hide: export %s=1"
```

Reply:
@jedevc wouldn't "see full trace" feel slightly inaccurate since it sends you to a setup, and then technically you won't see *that* particular trace?

- As an alternative: "Setup tracing at https://dagger.cloud/traces/setup. To hide, export NOTHANKS=1" (78)

Reply:
(I updated my proposal based on this)